UIDokan/Sources/UIDokan/DesignSystem/DesignSystem.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  File.swift
+//  
+//
+//  Created by ziad on 05/07/2022.
+//
+
+import Foundation
+import UIKit
+ public struct DesignSystem{
+
+
+     //MARK: - Colors
+     public enum Colors: String{
+         // Primary
+         case blueOcean = "Blue Ocean"
+         case orangeFresh = "Orange Fresh"
+         case navyBlack = "Navy Black"
+         case pureWhite = "Pure White"
+         // Secondary
+         case earthGreen = "Earth Green"
+         case redVelvet = "Red Velvet"
+         case darkGrey = "darkGrey"
+         case halfGrey = "Half Grey"
+         case softGrey = "Soft Grey"
+         case offGrey = "Off Grey"
+         // Label
+         case offBlue = "Off Blue"
+         case offGreen = "Off Green"
+         case offRed = "Off Red"
+
+         public var color: UIColor{
+             switch self {
+             case .blueOcean:
+                 return UIColor(named: self.rawValue)!
+             case .orangeFresh:
+                 return UIColor(named: self.rawValue)!
+             case .navyBlack:
+                 return UIColor(named: self.rawValue)!
+             case .pureWhite:
+                 return UIColor(named: self.rawValue)!
+             case .earthGreen:
+                 return UIColor(named: self.rawValue)!
+             case .redVelvet:
+                 return UIColor(named: self.rawValue)!
+             case .darkGrey:
+                 return UIColor(named: self.rawValue)!
+             case .halfGrey:
+                 return UIColor(named: self.rawValue)!
+             case .softGrey:
+                 return UIColor(named: self.rawValue)!
+             case .offGrey:
+                 return UIColor(named: self.rawValue)!
+             case .offBlue:
+                 return UIColor(named: self.rawValue)!
+             case .offGreen:
+                 return UIColor(named: self.rawValue)!
+             case .offRed:
+                 return UIColor(named: self.rawValue)!
+             }
+         }
+     }
+
+     // MARK: - TypoGraphy
+     public enum TypoGraphy{
```

Comment:
- Same point as above. I prefer UIFont extension for a shorter usage.
#### Example: 
```swift
textField.font = .body
```
- Enum names should start with small case letter. 
- Please add only fonts we need and we should extend it on the run.